app/repository/pg/user.go

```diff
@@ -14,7 +14,7 @@ import (
 type Repo struct{ sql.DB }
 
 // NewRepo will initialise new instance of Repo
-func NewRepo(opt cfg.Options) *Repo {
+func NewRepo(opt *cfg.Options) *Repo {
```

Comment:
Why we are accepting opt argument via pointer?

Reply:
Victor has a point. We could accidentally change config fields while using the pointer receiver. That is worse case scenario. But [when in doubt, use a pointer receiver. ](https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#receiver-type)

Reply:
No, let's use another rule - need mutations (to change the value) - use pointer.
In other cases - please don't use it.

Reply:
> No, let's use another rule - need mutations (to change the value) - use pointer. In other cases - please don't use it.

Thank you, this rule makes the decision to use pointers much easier.
Fixed